content/300-guides/050-database/850-planetscale.mdx

```diff
@@ -225,6 +225,66 @@ model User {
 
 For a more detailed example, see the [Getting Started guide for PlanetScale](/getting-started/setup-prisma/add-to-existing-project/relational-databases/introspection-typescript-planetscale).
 
+## How to use the PlanetScale serverless driver with Prisma (Preview)
+
+The [PlanetScale serverless driver](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) provides a way of communicating with your database and executing queries over HTTP. Prisma by default uses TCP.
+
+You can use Prisma along with the PlanetScale serverless driver using a [driver adapter](/concepts/components/drivers) <span class="concept"></span>. A driver adapter allows you to use a different database driver, from the default Prisma provides, to communicate with your database.
+
+<Admonition>
+
+This feature is available in Preview from Prisma versions 5.4.2 and later.
+
+</Admonition>
+
+To get started, enable the `driverAdapters` Preview feature flag:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+```
+
+Generate Prisma Client:
+
+```sh
+npx prisma generate
+```
+
+Ensure you update the host value in your connection string to `aws.connect.psdb.cloud`. You can learn more about this [here](https://planetscale.com/docs/tutorials/planetscale-serverless-driver#add-and-use-the-planetscale-serverless-driver-for-javascript-to-your-project).
+
+```bash
+DATABASE_URL='mysql://johndoe:strongpassword@aws.connect.psdb.cloud/clear_nightsky?sslaccept=strict'
+```
+
+Install the Prisma adapter for PlanetScale, PlanetScale serverless driver and `undici` packages:
+
+```sh
+npm install @prisma/adapter-planetscale @planetscale/database undici
+```
+
+> Prisma ORM supports Node.js 16.13 and up. In Node.js 18 and up, `undici` is not needed. See [System requirements](/reference/system-requirements)
```

Comment:
How do the system requirements help with the undici requirement? 
(They would make sense with the "Prisma ORM supports Node ... and up" sentence)

```suggestion
> Prisma ORM supports Node.js 16.13 and up (See [System requirements](/reference/system-requirements)). In Node.js 18 and up, `undici` is not needed. 
```
But even the suggested sentence does not explain _why_ it is not needed and what it is needed for in the first place.

Also: Should this be an admonition, and not a quote?